graylog2-web-interface/src/components/common/Select.jsx

```diff
@@ -72,28 +73,24 @@ const clearIndicator = (base) => ({
   padding: '5px',
 });
 
-const multiValue = (base) => ({
+const multiValue = ({ theme }) => (base) => ({
   ...base,
-  backgroundColor: '#ebf5ff',
-  color: '#007eff',
-  border: '1px solid rgba(0,126,255,.24)',
+  border: `1px solid ${theme.colors.variant.lighter.info}`,
 });
 
-const multiValueLabel = (base) => ({
+const multiValueLabel = ({ theme }) => (base) => ({
   ...base,
-  color: 'unset',
   paddingLeft: '5px',
   paddingRight: '5px',
+  fontSize: theme.fonts.size.body,
```

Comment:
I like the new, larger font size, but we should adjust the select height (part of `controlNormal`) in the same step, right now the height of the select changes, when selecting e.g. a stream. 

Changing the font size is maybe something for another PR, since we should update the height of the time range selects in the same step as well.

Reply:
Switched this to use `size.small` so it'll render around 12px and then in the future we can determine if we want to modify the heights and font sizes of this along with the time range selector... as well as maybe move the time range selector over to using this Select component

Reply:
sounds good to me!